samples/sp_whoIsActive/src/sql/cpuDelta.sql

```diff
@@ -0,0 +1,6 @@
+EXEC sp_WhoIsActive 
```

Comment:
For all these queries I've found it's good to check for existence of the proc and if it doesn't exist, output results that indicate this. That way thing don't look "broken" before someone executes the "create sqp_whoisactive" task. doing an `IF EXISTS ... ELSE` works wonders for perceived reliability